include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -9088,6 +9333,34 @@
         }
       ]
     },
+    {
+      "category" : "ValueEnum",
+      "kind" : "FPDenormMode",
```

Comment:
Adding a new operand kind like this will break the SPIRV-Tools build.
I plan to prepare a patch to SPIRV-Tools to absorb this change.

This is the kind of error I get when rebasing this change against master and rebuilding SPIRV-Tools:


In file included from ...spirv-tools/source/operand.cpp:36:
third_party/spirv-tools/operand.kinds-unified1.inc:703:172: error: ‘SPV_OPERAND_TYPE_FPDENORM_MODE’ was not declared in this scope; did you mean ‘SPV_OPERAND_TYPE_MEMORY_MODEL’?
  703 |   {"FunctionDenormModeINTEL", 5823, 1, pygen_variable_caps_FunctionFloatControlINTEL, 1, pygen_variable_exts_SPV_INTEL_float_controls2, {SPV_OPERAND_TYPE_LITERAL_INTEGER, SPV_OPERAND_TYPE_FPDENORM_MODE}, 0xffffffffu, 0xffffffffu},
      |                                                                                                                                                                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                                                                                                                                                            SPV_OPERAND_TYPE_MEMORY_MODEL
third_party/spirv-tools/operand.kinds-unified1.inc:724:179: error: ‘SPV_OPERAND_TYPE_FPOPERATION_MODE’ was not declared in this scope; did you mean ‘SPV_OPERAND_TYPE_EXECUTION_MODE’?
  724 |   {"FunctionFloatingPointModeINTEL", 6080, 1, pygen_variable_caps_FunctionFloatControlINTEL, 1, pygen_variable_exts_SPV_INTEL_float_controls2, {SPV_OPERAND_TYPE_LITERAL_INTEGER, SPV_OPERAND_TYPE_FPOPERATION_MODE}, 0xffffffffu, 0xffffffffu},
      |                                                                                                                                                                                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                                                                                                                                                                   SPV_OPERAND_TYPE_EXECUTION_MODE
third_party/spirv-tools/operand.kinds-unified1.inc:1229:4: error: ‘SPV_OPERAND_TYPE_FPDENORM_MODE’ was not declared in this scope; did you mean ‘SPV_OPERAND_TYPE_MEMORY_MODEL’?
 1229 |   {SPV_OPERAND_TYPE_FPDENORM_MODE, ARRAY_SIZE(pygen_variable_FPDenormModeEntries), pygen_variable_FPDenormModeEntries},
      |    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |    SPV_OPERAND_TYPE_MEMORY_MODEL
third_party/spirv-tools/operand.kinds-unified1.inc:1230:4: error: ‘SPV_OPERAND_TYPE_FPOPERATION_MODE’ was not declared in this scope; did you mean ‘SPV_OPERAND_TYPE_EXECUTION_MODE’?
 1230 |   {SPV_OPERAND_TYPE_FPOPERATION_MODE, ARRAY_SIZE(pygen_variable_FPOperationModeEntries), pygen_variable_FPOperationModeEntries},
      |    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |    SPV_OPERAND_TYPE_EXECUTION_MODE

Reply:
I have a pending patch for SPIRV-Tools.  https://github.com/KhronosGroup/SPIRV-Tools/pull/4116  
That will have to land before this patch (to prevent the SPIRV-Tools build from breaking).

Reply:
Yeah, it also required changes in the headers builder itself.

Reply:
I've landed the SPIRV-Tools change, so we're clear to land this one, from my perspective.